source/reference/minio-mc/mc-replicate-status.rst

```diff
@@ -131,3 +131,19 @@ Use :mc:`mc replicate status` to show bucket replication status:
 - Replace :mc-cmd:`ALIAS <mc replicate status ALIAS>` with the :mc:`alias <mc alias>` of the MinIO deployment.
 
 - Replace :mc-cmd:`PATH <mc replicate status ALIAS>` with the path to the bucket or bucket prefix.
+
+Behavior
+--------
+
+Removed and Re-added ARNs
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionchanged:: mc RELEASE.2023-03-20T17-17-53Z
+
+ARNs no longer part of replication configurations do not display with the standard output of this command.
```

Comment:
This sentence seems like it could be simplified? Perhaps along the lines of "The standard output of this command does not display ARNs that have been removed from a replication configuration."

Reply:
I like your suggestion with a slight tweak to avoid passive.